.gitignore

```diff
@@ -36,7 +36,6 @@ prepros.json
 .tmp
 *.diff
 *.err
-*.lock
```

Comment:
This was actually a change that wouldn't really make sense here but rather in the framework repo. I'll commit a reversal. This would break core's semver patch functionality but on framework I would recommend removing that line but also never actually committing the lock file to the Github repo (assuming you guys are using yarn).